src/amo/components/Collection/index.js

```diff
@@ -111,12 +115,11 @@ export class CollectionBase extends React.Component<Props> {
       collectionChanged = true;
     }
 
-    // When switching into edit mode, refresh the collection add-ons.
-    if (this.props.editing && !nextProps) {
-      addonsPageChanged = true;
-    }
+    const collectionHasAddons = (collection
```

Comment:
Nit: There's not really a need for the parentheses here.

Reply:
oops.. yep...i forgot to remove these..